tests/integration/targets/aci_vrf_fallback_route_group/tasks/main.yml

```diff
@@ -0,0 +1,411 @@
+    
+    - name: Sort fallback members for fallback group
+      ansible.builtin.set_fact:
+        sorted_child_members: "{{ nm_create_vrf_fbrg.current[0].fvFBRGroup.children | selectattr('fvFBRMember', 'defined') | map(attribute='fvFBRMember.attributes') | list | sort(attribute='rnhAddr') }}"
+
+    - name: Verify that vrf_fbrg has been created correctly with the correct attribute values
+      ansible.builtin.assert:
+        that:
+          - cm_create_vrf_fbrg is changed
+          - cm_create_vrf_fbrg.previous == []
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - cm_sorted_child_members.0.rnhAddr == "1.1.1.1"
+          - cm_sorted_child_members.1.rnhAddr == "2.2.2.2"
+          - cm_sorted_child_members.2.rnhAddr == "4.4.4.4"
+          - cm_sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - nm_create_vrf_fbrg is changed
+          - nm_create_vrf_fbrg.previous == []
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - sorted_child_members.0.rnhAddr == "1.1.1.1"
+          - sorted_child_members.1.rnhAddr == "2.2.2.2"
+          - sorted_child_members.2.rnhAddr == "4.4.4.4"
+
+    - name: Create vrf fallback route again (idempotency)
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *fbrg_info
+      register: nm_create_vrf_fbrg_again
+    
+    - name: Verify that nm_create_vrf_fbrg_again is not changed (idempotency)
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg_again is not changed
+          - nm_create_vrf_fbrg_again.current == nm_create_vrf_fbrg.current
+
+    - name: Create a new vrf Fallback Route Group with minimum required config - Normal mode   
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group_2 
+        state: present
+      register: nm_create_vrf_fbrg_2
+
+    - name: Verify that nm_create_vrf_fbrg_2 is created with correct attributes
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg_2 is changed
+          - nm_create_vrf_fbrg_2.previous == []
+          - nm_create_vrf_fbrg_2.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group_2"
+          - nm_create_vrf_fbrg_2.current.0.fvFBRGroup.children is not defined
+
+    - name: Create a new vrf Fallback Route Group with minimum required config - Check mode  
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group_2 
+        state: present
+      check_mode: true
+      register: cm_create_vrf_fbrg_2
+
+    - name: Verify that nm_create_vrf_fbrg_2 is created with correct attributes
```

Comment:
this is check mode assert and you are not mentioning in the task and are not asserting proposed